astore/rpc/auth.proto

```diff
@@ -28,6 +28,7 @@ message TokenResponse {
 
 message HostCertificateRequest {
   bytes hostcert = 1; // The public key of the host that will be returned as signed by the CA
+  repeated string hosts = 2; // A list of DNS names you wish for the host to have.
```

Comment:
this can also be IP addresses, right? nit: edit the comment.

Reply:
SSH CA's have the same limitations as TLS CA's where they can only trust dns names. IP's are invalid in principals (actually they're just a NoOp). SSHing into a box with valid certificates will always provoke a trust this host prompt.